client/js/components/procedure/Shared/SimilarStatementSubmitters/SimilarStatementSubmitters.vue

```diff
@@ -309,6 +311,27 @@ export default {
         })
     },
 
+    deleteEntry (index) {
+      this.updateStatement({
+        id: this.statementId,
+        relationship: 'similarStatementSubmitters',
+        action: 'remove',
+        value: {
+          id: this.listEntries[index].id,
+          type: 'SimilarStatementSubmitter'
+        }
+      })
+
+      this.listEntries.splice(index, 1)
+
+      if (this.isRequestFormPost === false) {
+        this.deleteSimilarStatementSubmitter()
+      }
+      if (this.isRequestFormPost) {
```

Comment:
nitpick: add empty line between ifs

```suggestion
      
      if (this.isRequestFormPost) {
```

Reply:
e13ad3e588c43000eb5bbcbd90664bc9919e7c04